src/app/forms/templates/test-records/create-master.template.ts

```diff
@@ -27,6 +31,17 @@ const groups1and2Template: Record<string, FormNode> = {
   reasonForCreation: reasonForCreationSection
 }
 
+const group10Template: Record<string, FormNode> = {
```

Comment:
Does it make sense to have a const for the group10 template if it's not being reused anywhere?

Reply:
Moved the group 10 template contents to be under the trl section